### DISCUSSION TITLE
Is it possible to use output: "export" in nextJS with Outstatic ?

### DISCUSSION THREAD

**Discussion by hoan2222** [avitorio/outstatic#227] [Q&A]
I am just new to Outstatic and am wondering if a static export is possible? 
Currently I am getting the following error in the (cms)/outstatic/[[..ost]]/page  : 

**Page "/(cms)/outstatic/[[...ost]]/page" is missing param "/outstatic" in "generateStaticParams()", which is required with "output: export" config.**

Does anyone has a Solution for that ? 

My page.jsx looks like this: 

```javascript
/(cms)/outstatic/[[...ost]]/page.jsx

import 'outstatic/outstatic.css'
import { Outstatic } from 'outstatic'
import { OstClient } from 'outstatic/client'
import { getCollections, getDocumentSlugs } from 'outstatic/server'



export async function generateStaticParams() {
    const posts = getCollections()
    console.log(posts.map((slug) => ({ slug })))
    return posts.map((slug) => ({ slug }))
}


export default async function Page({ params }) {
    const ostData = await Outstatic()
    console.log(params)
    return <OstClient ostData={ostData} params={params} />
}
``` 
My Dependencies: 

   "framer-motion": "^11.1.7",
    "next": "^14.2.3",
    "outstatic": "^1.4.8",
    "react": "^18",
    "react-dom": "^18"

**Comment by avitorio** [comment]
Hey, 
Sorry for the late reply.
That path and file are for the dashboard. That getStaticParams call shouldn't exist there. 

Please, read the docs and make the page.tsx file the same as we have it there.

Let me know if this works.

**Reply by hoan2222** [reply]
Hey,

yes originally I did it exactly as described in the docs. Then I am running in to this error:

**Error: Page "/(cms)/outstatic/[[...ost]]/page" is missing exported function "generateStaticParams()", which is required with "output: export" config.**

That's why I tried implementing the "generateStaticParams()" function...

Is it generally possible to use Outstation with _output: 'export'_ configured in next.config.js ? 

Thanks for your response:)

**Reply by avitorio** [reply]
I haven't tried it. Are you able to build the project normally without getStaticParams and the export option?

**Reply by avitorio** [reply]
"/(cms)/outstatic/[[...ost]]/page" shouldn't be part of your static build. Are you able to ignore it?

**Reply by hoan2222** [reply]
I am trying to deploy to GitHub pages, so sadly no. 
I thought  static export would be possible with "generateStaticParams()". 

Am I wrong ?

**Reply by avitorio** [reply]
The /outstatic dashboard shouldn't be a part of your export. You won't be able to access it on GitHub pages.

**Reply by bchristie** [reply]
@hoan2222 May want to check out https://github.com/avitorio/outstatic/discussions/237. At the very least offers a work-around.
